modules/serverless-rn-1-32-0.adoc

```diff
@@ -0,0 +1,69 @@
+
+* You can now use the gRPC health probe for `net-kourier` deployments. The `net-kourier-controller` now uses a standard Kubernetes gRPC health probe for both readiness and liveness, replacing its previous use of exec and custom commands. This simplifies the process and aligns it with Kubernetes standards. Additionally, the timeoutSeconds value has been adjusted from 100 milliseconds to 1 second to ensure more reliable probe responses.
+
+* The new trigger filters feature is now available as a Technology Preview. The new trigger filters are now enabled by default. It allows users to specify a set of filter expressions, where each expression evaluates to either true or false for each event.
+
+* Knative Eventing now offers support for data in transit encryption (Eventing TLS) as a developer preview. You can configure Knative Eventing components to expose HTTPS addresses as well as add user-provided CA trust bundles to clients. 
+
+* {ServerlessProductName} now supports custom OpenShift CA bundle injection for system components. For more information, see link:https://docs.openshift.com/container-platform/latest/networking/configuring-a-custom-pki.html#certificate-injection-using-operators_configuring-a-custom-pki[Configuring a custom PKI].
+
+* You can now use the Custom Metrics Autoscaler (KEDA) operator to autoscale Knative Eventing sources for Apache Kafka (KafkaSource), both up and down. This functionality is available as a developer preview, offering enhanced scalability and efficiency for Kafka-based event sources within Knative Eventing.
+
+* You can now explore the Knative Eventing monitoring dashboards directly within the *Observe* tab of the developer view in the OpenShift Developer Console. 
+
+* The support for EventTypes `v1beta1` in Knative shipped is deprecated in {ServerlessProductName} 1.32. In {ServerlessProductName} 1.32, the Knative CLI utilizes the EventType `v1beta2` API to facilitate the new reference model. The `kn` is not backward compatible with the EventType API v1beta1 in previous releases, specifically limited to the `kn eventtypes` sub-commands group. Therefore, it is recommended to use a matching `kn` version for the best user experience.  
+
+[id="fixed-issues-1-32-0_{context}"]
+== Fixed issues
+
+* The default CPU limit is now increased for `3scale-kourier-gateways` from `500m` to `1`. When more than 500 Knative Service instances are created, it could lead to readiness and liveness probe failures in the 3scale-kourier-gateways pod due to CPU resource exhaustion. This adjustment aims to mitigate such failures and ensure smoother operation even under heavy loads.
+
+[id="known-issues-1-32-0_{context}"]
+== Known issues
+
+* Due to different mountpoint permissions, direct upload on a cluster build is not working on `ppc64le` and `s390x`.
```

Comment:
```suggestion
* Due to different mount point permissions, direct upload on a cluster build does not work on `ppc64le` and `s390x`.
```

Reply:
mount point word usage: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#m

Reply:
Maybe it's just me but direct upload of what does not work on `ppc64le` and `s390x`?

Reply:
Corrected! It is IBM zSystems (s390x) and IBM Power (ppc64le)